### PR TITLE
TW 802 Manage team - invites sent

### PR DESCRIPTION
**Pull request by HeatherStenson**

### CODE REVIEW

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 1]
```diff
@@ -57,56 +58,64 @@ You can invite collaborators, remove existing team members, and update user perm
 
 ### Inviting users
 
-You can invite users to your Postman team by navigating to [Manage Team](https://go.postman.co/settings/team/members) and selecting **Invite People**. Enter the email addresses of the individuals you'd like to invite and select **Add**. Assign these users one or more [team roles](/docs/collaborating-in-postman/roles-and-permissions/#team-roles), then select **Send Invites**.
+1. From your [team dashboard](https://go.postman.co/settings/team/members), select **Invite People**.
+1. Enter the email addresses of the individuals you want to invite.
 
-<img alt="Invite users to team" src="https://assets.postman.com/postman-docs/team-invite-people-v9.1.0.jpg" width="350px"/>
+    > You can also upload a .txt or .csv file with the email addresses of the people you want to invite. To do so, select **Add from file**.
```
Comment (petewitcher) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:64]: I think file extensions would be a good candidate for the next style guide review round. The LC mostly uses all caps ("JSON file") but our style guide points to the MS style guide which advises all lowercase for file extensions. Personally, I'd change .txt and .csv to .TXT and .CSV for relative consistency but it's your call.

**Review by petewitcher — APPROVED** [review]
LGTM! I commented on the .txt and .csv file extensions but it's not a show stopper IMO.

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 2]
```diff
@@ -57,56 +58,64 @@ You can invite collaborators, remove existing team members, and update user perm
...
 
-You can review invite requests team members have submitted in your [team dashboard](https://go.postman.co/settings/team/members) under **Suggestions**. To respond to a request, hover over the individual and select **Invite** or the dismiss suggestion icon <img alt="Close icon" src="https://assets.postman.com/postman-docs/icon-close.jpg#icon" width="16px">. If an invite has been sent to a suggested user, but they haven't yet joined your team, you'll have the option to resend it, copy the invite link, or dismiss it.
+You can review also pending invite requests that team members have submitted in your [team dashboard](https://go.postman.co/settings/team/members) under **Invites sent**. To resend an invite, hover over the individual's name and select **Resend Invite**. To copy the invite link, select the copy invite link icon <img alt="Copy invite link icon" src="https://assets.postman.com/postman-docs/icon-copy-v9.jpg#icon" width="15px">. To delete the invite, select the revoke invite icon <img alt="Revoke invite icon" src="https://assets.postman.com/postman-docs/icon-close.jpg#icon" width="16px">.
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:93]:
```suggestion
You can also review pending invite requests that team members have submitted in your [team dashboard](https://go.postman.co/settings/team/members) under **Invites sent**. To resend an invite, hover over the individual's name and select **Resend Invite**. To copy the invite link, select the copy invite link icon <img alt="Copy invite link icon" src="https://assets.postman.com/postman-docs/icon-copy-v9.jpg#icon" width="15px">. To delete the invite, select the revoke invite icon <img alt="Revoke invite icon" src="https://assets.postman.com/postman-docs/icon-close.jpg#icon" width="16px">.
```

**Review by mark-anthony-postman — APPROVED** [review]
Changes look great! 🙌 Made a couple o' suggestions you can consider 😀

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 3]
```diff
@@ -57,56 +58,64 @@ You can invite collaborators, remove existing team members, and update user perm
...
 
 <img alt="Confirmation for removing a team member" src="https://assets.postman.com/postman-docs/remove-user-from-team-v9.2.jpg" width="400px"/>
 
-> Note that when you remove a member from your team, this doesn't automatically reduce the number of paid slots for your team unless the member was added by [Auto-Flex](/docs/administration/billing/#using-auto-flex) during the current Auto-Flex cycle. If you want to reduce the number of paid slots on your team, a member of your team with the [Billing role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) can [edit your plan](/docs/administration/billing/#changing-your-plan).
+> Note that when you remove a member from your team, this doesn't automatically reduce the number of paid seats for your team unless the member was added by [Auto-Flex](/docs/administration/billing/#utilizing-auto-flex) during the current Auto-Flex cycle. If you want to reduce the number of paid seats on your team, a member of your team with the [Billing role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) can [edit your plan](/docs/administration/billing/#changing-your-plan).
```
Comment (HeatherStenson) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:107]:
```suggestion
> Note that when you remove a member from your team, this doesn't automatically reduce the number of paid seats for your team unless the member was added by [Auto-Flex](/docs/administration/billing/#using-auto-flex) during the current Auto-Flex cycle. If you want to reduce the number of paid seats on your team, a member of your team with the [Billing role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) can [edit your plan](/docs/administration/billing/#changing-your-plan).
```

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 4]
```diff
@@ -125,8 +134,8 @@ You can manage your team size or upgrade your plan at any time during your team'
 
 If you want to change your team's size, you have two different options:
 
-* **Change the number of paid slots** - Team members with the Billing role can change your team's number of paid slots in the [billing dashboard](https://go.postman.co/billing/overview).
-* **Enable Auto-Flex** - Your team can invite more users without having to pay in advance for more seats with [Auto-Flex](/docs/administration/billing/#using-auto-flex).
+* **Change the number of paid seats** - Team members with the Billing role can change your team's number of paid seats in the [billing dashboard](https://go.postman.co/billing/overview).
+* **Turn on Auto-Flex** - Your team can invite more users without having to pay in advance for more seats with [Auto-Flex](/docs/administration/billing/#utilizing-auto-flex).
```
Comment (HeatherStenson) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:138]:
```suggestion
* **Turn on Auto-Flex** - Your team can invite more users without having to pay in advance for more seats with [Auto-Flex](/docs/administration/billing/#using-auto-flex).
```

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 5]
```diff
@@ -57,56 +58,64 @@ You can invite collaborators, remove existing team members, and update user perm
 
 ### Inviting users
 
-You can invite users to your Postman team by navigating to [Manage Team](https://go.postman.co/settings/team/members) and selecting **Invite People**. Enter the email addresses of the individuals you'd like to invite and select **Add**. Assign these users one or more [team roles](/docs/collaborating-in-postman/roles-and-permissions/#team-roles), then select **Send Invites**.
+1. From your [team dashboard](https://go.postman.co/settings/team/members), select **Invite People**.
+1. Enter the email addresses of the people you want to invite.
 
-<img alt="Invite users to team" src="https://assets.postman.com/postman-docs/team-invite-people-v9.1.0.jpg" width="350px"/>
+    > You can also upload a .txt or .csv file with the email addresses of the people you want to invite. To do so, select **Add from file**.
```
Comment (HeatherStenson) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:64]:
```suggestion
    > You can also upload a text or CSV file with the email addresses of the people you want to invite. To do so, select **Add from file**.
```

Thanks for calling this out @petewitcher! I think I was misremembering how the decision about zip files went and was extrapolating from that 😅 Per the Google guidance, which is what we made the zip file decision based on, this should be "text file" and "CSV file". I'll update this PR accordingly and will add this issue to our backlog.

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 6]
```diff
@@ -46,8 +47,8 @@ Postman provides a variety of options to customize your team's experience, from
 
 Once you've created your Postman team, you can take the following steps to enable an efficient onboarding process for end users:
 
-* Use the [Onboarding checklist](/docs/administration/onboarding-checklist/) to ensure that Postman is accessible on employee devices and functions as expected.
-* [Enable team discovery](/docs/collaborating-in-postman/working-with-your-team/enabling-team-discovery/) to allow fellow Postman users from your organization to request to join your team.
+* Use the [Onboarding checklist](/docs/administration/onboarding-checklist/) to ensure that Postman is available on employee devices and that it functions as expected.
```
Comment (jkonrath-postman) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:50]: Is it weird to use "employee" here? Just thinking, they could be students, volunteers, external users, etc.

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 7]
```diff
@@ -57,56 +58,78 @@ You can invite collaborators, remove existing team members, and update user perm
...
+
+<img alt="Review invites sent" src="https://assets.postman.com/postman-docs/v10/team-manage-invite-requests-v10.jpg"/>
+
+#### Suggestions
 
-To copy an active invite URL, hover over a row and select the copy icon <img alt="Copy link icon" src="https://assets.postman.com/postman-docs/icon-copy-v9.jpg#icon" width="15px">.
+If a team member who isn't a Developer inviting another Developer on a Free, Basic, or Professional plan or a Team Admin on any plan, you or another team member with the Admin or Super Admin role must review the invite suggestion. You can review these invite suggestions in your [team dashboard](https://go.postman.co/settings/team/members) under **Suggestions**.
```
Comment (jkonrath-postman) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:95]: Is there any way to flip this sentence or break it up to make it less confusing? I was reading and got up to "inviting" and wondering if it should be "invites" and then kept reading and eventually got it.

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 8]
```diff
@@ -99,7 +99,7 @@ To invite partners to a Partner Workspace, do the following:
 1. (Workspace Admins and Partner Managers only) Select **Make them Partner Lead** if you'd like them to also be able to manage and invite partners within the workspace.
 1. Select **Invite Partners**.
 
-Partner invites must be accepted by the email ID you provide when inviting the individual. If a partner would like to use a different email ID, or if an invite was mistakenly sent, you can [revoke the invite](/docs/administration/managing-your-team/managing-your-team/#managing-invitations) and send a new one.
+Partner invites must be accepted by the email ID you provide when inviting the individual. If a partner would like to use a different email ID, or if an invite was mistakenly sent, you can [revoke the invite](/docs/administration/managing-your-team/managing-your-team/#managing-invites) and send a new one.
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:102]: "email ID" sure looks weird. Should we just use email? email address? Discuss next cycle?

**Review by jkonrath-postman — APPROVED** [review]
A couple of little things, but otherwise LGTM!

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md (hunk 8 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:102]: Thanks for catching. I'm changing this to "email address".

File: src/pages/docs/administration/managing-your-team/managing-your-team.md (hunk 7 above)
Comment (HeatherStenson) [reply src/pages/docs/administration/managing-your-team/managing-your-team.md:95]: Yeah, this one's a 👹 After a little rejiggering, I've come up with:

> If the team member sending an invite isn't a Developer inviting another Developer (on Free, Basic, or Professional plans) or if they aren't a Team Admin (on any plan), you or another team member with the Admin or Super Admin role must review the invite suggestion.

Thoughts?

**Review by ArjunSingh-PM — APPROVED** [review]
The changes look great! Thanks, Heather!

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Managing your team"
 order: 121
-updated: 2022-06-01
+updated: 2022-12-13
```
Comment (HeatherStenson) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:4]:
```suggestion
updated: 2022-12-19
```

**Pull request merged**